Exercise02-Basics/Svetlin-Trayanski/BASICS03-009.js

```diff
@@ -0,0 +1,7 @@
+/*Calculate and print on the console the perimeter of a triangle.*/
+
+let a = 3;
+let b = 4;
+let c = 6;
+let P = a + b + c
+console.log(P)
```

Comment:
local variables, defined with `let` should be declared with camelCase naming convention

```js
let perimeter = a + b + c;       
```